obs-packaging/linux-container/update.sh

```diff
@@ -46,9 +46,10 @@ RELEASE=$(get_obs_pkg_release "${PROJECT_REPO}")
 
 kernel_sha256=$(curl -L -s -f ${KR_SHA} | awk '/linux-'${VERSION}'.tar.xz/ {print $1}')
 
-# Generate the kernel config file
-KERNEL_ARCH=$(go get github.com/kata-containers/tests && $GOPATH/src/github.com/kata-containers/tests/.ci/kata-arch.sh --kernel)
-cp "${SCRIPT_DIR}/../../kernel/configs/${KERNEL_ARCH}_kata_kvm_${KR_LTS}.x" config
+# Copy the kernel config files for all architecture
+mkdir -p configs
+readonly configs_dir="kernel/configs"
+find "${SCRIPT_DIR}/../../${configs_dir}" -name "*_kata_kvm_${KR_LTS}.x" -exec tar --transform="s,${configs_dir},kata-configs," -czf kata-configs.tar.gz {} +
```

Comment:
Wow ;)

Reply:
I recon the `find ... -exec` syntax is not so obvious to understand for those not familiar with it :P